daemon/mgr/container_types.go

```diff
@@ -17,3 +20,29 @@ type ContainerRemoveOption struct {
 	Volume bool
 	Link   bool
 }
+
+// Container represents the container instance in runtime.
+type Container struct {
+	sync.Mutex
+	meta *types.ContainerInfo
```

Comment:
Why not just use `types.ContainerInfo` directly?
Is it related to the pointer which could save some memory?